### DISCUSSION TITLE
Time to go to gitlab, next Month is two-factor authentication  obligation, manny users wil go out MS-Github

### DISCUSSION THREAD

**Discussion by blackcrack** [neutrinolabs/xrdp#2894] [Q&A]
Hi,

in sence of security is mostly ein stalking-horse for become real Datas about Users, manny Users be here in
a Aliasaccount and want hide the real name.. also like in the old times, this want MS remove and want become over Telephone-data and number the Real User Name (in the most Eurolands is the user mostly to 95% regged over TelNo.) and the whole europol be in possibility for become the realname on Phonenumber, i guess MS have a Man in a Grouping and becomes therewith also over the nubers the real name of the Users of the Phone.. i guess this is the real reason why the two-factor authentication ... and the most make it more easy and take easy without thinking the handy, photograph therewith the code and send it with the phonenumber where is writing in and therewith is open the way informations about the user ..

because this, should the Project go fully to gitlab.

think about...

best regards
Blacky

**Comment by metalefty** [comment]
Feel free to migrate YOUR project to GitLab!

**Reply by blackcrack** [reply]
Hi @metalefty ,

i have remove all project from my account like you can see and the removed all notifications from github,
if i have a problem behind 19th for "savety" and prevent me from account things at Microsofts Github account for changing .. This " extortion" of "you have make what we want" in security, is a bad thing and not for really free Peoples, if MS as Commerial Factory now try to become more information about his customers and this bring in "in the Windows Coudservers" via Windows 356 for bind the Users hard to MS and to verwatch the whole software,
make the whole fast evelution crashed and bring down the free distributioning in the gray sence ;),
delete without questions technical helping tools and so on.. MS only benefits from OpenSource, 
if it's costs nothing or brings in profit but not in sense in free world, only in comercial world and view...

and this, "you have to take 2FA or you have a problem" is a extortion and the Users cant decide by self,
if "more security" or not.. they have to do ... of have a Problem from MS-side on Github.

> :warning: GitHub users are [now required](https://github.blog/2023-03-09-raising-the-bar-for-software-security-github-2fa-begins-march-13/) to enable two-factor authentication as an additional security measure. Your activity on GitHub includes you in this requirement. You will need to enable two-factor authentication on your account before January 19, 2024, or be restricted from account actions. [Button enable 2FA]

eh no, thank you.. M$ :exclamation: 



best regards to you @metalefty  :hugs:

**Comment by matt335672** [comment]
@blackcrack 

I suggest you look into 2FA in more detail. You don't have to use a mobile phone number for this - you can use a security key for example. That won't make you any less anonymous than you already are and it will greatly improve your own security.

**Comment by blackcrack** [comment]
@matt335672 , @metalefty 
Hi both Buddys ;)

there is no question, if use 2FA or not, there is a extortion, if you not use have you trubles..

now have i become a huge Mail from "GitHub" :  

```
Hey blackcrack!

This is a reminder that [we announced](https://github.blog/2022-05-04-software-security-starts-with-the-developer-securing-developer-accounts-with-2fa/) that we are requiring users contributing code on GitHub.com to enable two-factor authentication (2FA). You are receiving this notification because your account meets this criteria and will be required to enroll in 2FA by January 19th, 2024 at 00:00 (UTC).

Please see the below FAQ, or learn more about [2FA on GitHub Help.](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/about-two-factor-authentication)
What you need to know about the required 2FA initiative

We are enrolling GitHub users who manage or author code on GitHub. More information about our efforts to make 2FA adoption easy and safe can be found in [this blog post.](https://github.blog/2023-03-09-raising-the-bar-for-software-security-github-2fa-begins-march-13) This is a GitHub.com program, and unrelated to any organization or enterprise membership your account may have.
How will this affect my account?

On January 19th, 2024 at 00:00 (UTC) your account will be required to have 2FA for authentication. If you have not yet enrolled by that date, your ability to access GitHub.com will be limited until you finish the enrollment process.
How do I enroll in 2FA?

[Click here to get started!](https://github.com/settings/two_factor_authentication/setup/intro) Prior to January 19th, 2024 at 00:00 (UTC) you can follow the instructions [in our documentation](https://docs.github.com/articles/configuring-two-factor-authentication) to set up 2FA for your account. If you have not yet enrolled in 2FA by January 19th, 2024 at 00:00 (UTC), you will automatically be taken to the 2FA enrollment form the next time you access GitHub.com.
What forms of 2FA can I use?

We want you to have the most seamless experience with 2FA possible, so you can choose one or more of the following options:

    Security key
    GitHub Mobile
    Authenticator application (TOTP)
    Text messages (SMS)

You should set up at least two of these options, to ensure you always have access to your account. Head to https://github.com/settings/security to enroll more 2FA methods.
I already have 2FA enabled, do I need to do anything?

No, if you already have 2FA enabled before January 19th, 2024 at 00:00 (UTC), you don't need to take any additional actions. After January 19th, 2024 at 00:00 (UTC), you will no longer be able to unenroll from 2FA from your account, but you will be able to change the option you use for authenticating with 2FA. Additionally, you won't see any more banners on GitHub.com, and we won't email you about this anymore.
What happens to my PATs and SSH keys at the deadline?

Your PATs, SSH keys, and applications will all keep working after the deadline, regardless of your 2FA enrollment. PATs in particular are used extensively in important automation, and interruption there can cause outages in critical systems. However, when it is time to sign in to GitHub.com to create a new PAT or manage your account, you'll be required to enable 2FA before you can proceed.
What do I do if I lose my 2FA device?

GitHub strongly encourages the use of multiple second factor options. If you lose all of your second factors, recovery codes are the only way to access your account again. By saving your recovery codes, you'll be able to regain access.

Be sure to enable cloud backup for your authenticator app and save your recovery codes. Many phones and computers can be security keys as well - registering them with GitHub.com gives you additional, highly-secure 2FA methods.

For security reasons, GitHub Support may not be able to restore access to accounts with 2FA enabled if you lose your 2FA credentials and lose access to your account recovery methods.

More information about recovery codes can be found on GitHub Help at https://docs.github.com/articles/recovering-your-account-if-you-lose-your-2fa-credentials
Why is GitHub requiring 2FA?

Ensuring account security is a shared responsibility GitHub takes seriously. Strong authentication and the use of 2FA have been recognized as best practice for many years. We feel that GitHub has a duty to lead this push toward strong authentication as part of protecting the software supply chain.

To see this and other security events for your account, visit [your account security audit log.](https://github.com/settings/security-log)

If you run into problems, please contact support by visiting [the GitHub support page.](https://github.com/contact)

Thanks,
The GitHub Team
```

All+l in the sence of "Security and Savety" 
by the way, the SecurityKey is hast commes  later.. first has it want become the Automatism have the Telefonnumber of the users where i have try what's happens if i start the 2FA thing and i have broke it up because "only you Telephone number" have i stop this crap .. nope.. thanks.. i let not use me from microsoft if they want my datas and something else for make money..
this is the reason why i dont play this b..s.. Game not with.. more likely i make nothing anymore and hold my name as account only.. fullstop ... 

best regards and a good new year belated ;)
Blacky

**Reply by metalefty** [reply]
I don't stop quit using GitHub. Free free to do that. It's your desicion. Bye.
